docs/guides/cms/cloud_server_using_nextcloud.md

```diff
@@ -14,37 +14,37 @@ tags:
 - Server running Rocky Linux (you can install Nextcloud on any Linux distribution, but this procedure assumes you are using Rocky).
 - A high degree of comfort operating from the command line for installation and for configuration.
 - Knowledge of a command-line editor. Using `vi` for this example, but you can use your favorite editor if you have one.
-- This procedure covers the `.zip` file installation method. You can also install Nextcloud with a snap application.
+- This procedure covers the `.zip` file installation method. You can also install Nextcloud via a snap.
 - This procedure uses the Apache _sites enabled_ document (linked later) for directory setup.
```

Comment:
Vale really doesn't like "via" ... suggests to always use simple language. I think I'm going to switch this one back to "with" ... ;-)